update-docs.ps1

```diff
@@ -1,2 +1,8 @@
+# Usage: .\update-docs.ps1. This will regenerate the docs for all exercises.
+# Usage: .\update-docs.ps1 -o <exercise>. This will regenerate the docs for the specified exercise.
```

Comment:
Why `-o` out of curiosity? I would have expected `-e` for exercise.

Reply:
Well, that is how the underlying `configlet` binary expects its parameter.

In a future update, I plan to bring back the strongly-typed parameters, so you'll do `.\update-docs.ps1 -Exercise <exercise>` then.